### DISCUSSION TITLE
How to get slidesPerView index

### DISCUSSION THREAD

**Discussion by liho00** [nolimits4web/swiper#6690] [General]
For example i have a carousel with number of slides 100 and slidesPerView: 5

I want to get index 0,1,2,3,4 of each slides on the view when swiping from beginning to end, but how?

**Comment by ramsaysewell** [comment]
Hey @liho00,

A good solution to this depends on what you'd like the result to look like. Without knowing which platform you're using (react/vue/element), it's tricky to help with this as they all have their own limitations.

From the way you're writing your attributes however, I'd assume it's a jsx based framework like React so I'll assume it's React.

Reading the docs here: https://swiperjs.com/react#swiperslide-render-function, you'll find useful information regarding the attributes that you get to play with when rendering each slide. If you'd like to set specific styles or prop values to slides that are visible or active, you can use `isVisible` or `isActive`, respectively.

Hopefully this helps you out. If you can share some more information on what you're trying to do and what framework you're using, I'm sure we can help you out more!
